09 Environment/ENV_ProcedureForBranching.md

```diff
@@ -0,0 +1,14 @@
+## Procedure for at lave en branch
+
+* For at lave en ny branch skal du gå ind på det repository hvor du ville lave en ny branch
+
+* Derefter tryk på master branch
+
+![image](https://user-images.githubusercontent.com/71014091/118826678-fb555200-b8bb-11eb-8256-7ffc67d1af5b.png)
+
+
+Giv derefter branchen et navn som giver mening for hvad branchen skal bruges til
```

Comment:
"et navn, som giver mening for, hvad branchen".
Plus punktum til sidste i linje 5 samt 10.